tests/substrate.rs

```diff
@@ -266,7 +273,9 @@ impl Externals for MockSubstrate {
 
                 self.store.insert((self.vm.account, key), data);
 
-                Ok(None)
+                // The size of the pre-existing value is returned (if any).
+                // In this mock VM we just always assume that there was no value and return `None`.
+                Ok(Some(NONE_SENTINEL))
```

Comment:
Same as clear storage, just calculate the old length, it's not much effort.
```rust
let old_length = self.store.get)(&(self.vm.account, key)).map(|e| e.len()).unwrap_or(0);
// remove entry
return Ok(Some(Runtime:i32(old_length as i32));
```

Much nicer than return bugs stuff